app/controllers/catalog_controller.rb

```diff
@@ -63,7 +63,13 @@ def moab_storage_root
 
   # @return boolean
   def checksums_validated
-    return params[:checksums_validated].casecmp('true').zero? if params[:checksums_validated]
-    false
+    case params[:checksums_validated]
+    when TrueClass, FalseClass
+      params[:checksums_validated]
```

Comment:
codeclimate indicates that this branch is uncovered, which surprised me slightly given the modified test.  i can take another look at that to make sure i modified the relevant test case...

Reply:
🤷🏻‍♂️